thunder/core/langctxs.py

```diff
@@ -64,8 +64,17 @@ def reset_langctx(token: Any, /) -> None:
 
 # A helper for acquiring a method
 def resolve_method(id: Any, *args, **kwargs) -> Callable:
-    ctx: LanguageContext = get_langctx()
-    method: Callable = ctx.get_method(id, *args, **kwargs)
+    # I could combine the try/except statements but intentionally separated them for clarity
+    try:
+        ctx: None | LanguageContext = get_langctx()
+    except LookupError:
+        return None
+    try:
+        # AttributeError occurs when there is language context is None or when that context does not have the attribute
```

Comment:
This comment is a little confusing. If the language context is `None`, then won't that be caught by the previous case?

So should this now be:

```
# ctx.get_method throws an AttributeError when the context does not have the requested attribute, except
# for the prims language context, which always throws a ValueError
```

?

Reply:
Yes, you are right. Thanks for a clearer comment! Will update.